prism/src/main/java/org/apache/falcon/resource/AbstractEntityManager.java

```diff
@@ -172,6 +173,9 @@ protected static void checkColo(String colo) {
             for (String cluster : clusters) {
                 try{
                     Cluster clusterEntity = EntityUtil.getEntity(EntityType.CLUSTER, cluster);
+                    if(type.equals(EntityType.PROCESS)){
+                        EntityUtil.getEntity(EntityType.PROCESS,entity.getName());
```

Comment:
Hi,

The root cause of the issue was the process was present on the prism but was deleted from one of the colo's in distributed mode and we are getting a NPE for that colo when we do a status. 
Through this code I am checking in the configrationstore if that process actually exists at that colo.As for one of the clusters it will give me EntityNotRegisteredException, we are catching it and we will not get NPE.